piker/ui/_flows.py

```diff
@@ -416,6 +417,10 @@ def maxmin(
             if not slice_view.size:
                 mxmn = None
 
+            elif self.yrange:
```

Comment:
the bit where we choose to use the m4 yrange output if set instead of the normal "manual" sorting calls from prior.